app/controllers/api/v2/congress_controller.rb

```diff
@@ -1,21 +1,33 @@
 class Api::V2::CongressController < ApplicationController
 
   def zipcode_with_quiz
-    representatives = Representative.from_zip(params[:zipcode])
-    quiz = parse_quiz(params)
-    render json: {
-                    politicians: RepresentativeSerializer.new(representatives, {params: quiz}),
-                    summary_statistics: SummaryStatisticsSerializer.new(VoteSmartFacade.summary_statistics(representatives, quiz))
-                  }
+    if params[:aclu].nil?
+      render json: {error: 'please include quiz!'}, status: :bad_request 
+    elsif params[:zipcode].blank? 
+      render json: {error: 'please include zipcode!'}, status: :bad_request 
+    else 
+      representatives = Representative.from_zip(params[:zipcode])
+      quiz = parse_quiz(params)
+      render json: {
+                      politicians: RepresentativeSerializer.new(representatives, {params: quiz}),
+                      summary_statistics: SummaryStatisticsSerializer.new(VoteSmartFacade.summary_statistics(representatives, quiz))
+                    }
+      end
   end
 
   def state_with_quiz
-    representatives = Representative.from_state(params[:state])
-    quiz = parse_quiz(params)
-    render json: {
-                    politicians: RepresentativeSerializer.new(representatives, {params: quiz}),
-                    summary_statistics: SummaryStatisticsSerializer.new(VoteSmartFacade.summary_statistics(representatives, quiz))
-                  }
+    if params[:aclu].nil?
```

Comment:
We could potentially break this conditional up into a private method called something like serializer checks that takes an arg of zip or state since this code is very similar to zip with quiz.